src/Components/cards/card.jsx

```diff
@@ -32,18 +32,28 @@ function ProductCard() {
               <b class="discount-price">24% OFF</b>
             </div>
             <div class="card-button">
-              <button
-               onClick={()=>dispatch({type:"ADD_TO_CART",payload:items})}
+            {cart.filter((item_) => item_._id == item._id).length ? <button
```

Comment:
correct me if I'm wrong I don't think .length func needs to be used here you can just use cart.filter()? gotoCart : Addtocart